### DISCUSSION TITLE
Care to explain what dbt mean in README ?

### DISCUSSION THREAD

**Discussion by wenerme** [lightdash/lightdash#4061] [Q&A]
Sorry if this noob, but there are alot dbt in README, have no idea what dose that mean.

**Comment by TuringLovesDeathMetal** [comment]
Hi @wenerme 👋 dbt is a data transformation tool. To use Lightdash, you need to be using dbt as your data transformation layer. 

If you're not familiar with dbt, I'd recommend checking out dbt's getting started guide here: https://docs.getdbt.com/tutorial/setting-up 🙂
